plugin/src/main/kotlin/tanvd/grazi/ide/notification/Notification.kt

```diff
@@ -54,7 +54,9 @@ object Notification {
                                 state.nativeLanguage
                             }
 
-                            state.copy(enabledLanguages = state.enabledLanguages - state.missedLanguages, nativeLanguage = native)
+                            val enabledLanguages = state.enabledLanguages - state.missedLanguages
+                            state.copy(enabledLanguages = enabledLanguages, nativeLanguage = native,
+                                    enabledLanguagesAvailable = enabledLanguages.filter { it.jLanguage != null }.toSet())
```

Comment:
Let's add some sort of copy that will setup enabledLanguagesAvailable automatically

Reply:
Remove data class and add custom equals/hascode/copy methods